static/css/documentation.css

```diff
@@ -203,6 +203,10 @@ a img {
   text-decoration: none;
 }
 
+.bodyField {
```

Comment:
we tend not to use camel case classes in this project. how about body-field?

Reply:
[un-camel'd](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.css#diff-331344c9aef74ed7c8415f3632a7c619L206) and updated the [def table](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.erb#diff-1adfd305d8a0445e782d6f126c58d760L12) and [patch def table](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.erb#diff-bac15500a3095fd0a82601de2a06101eL12)